opnode/contracts/doc.go

```diff
@@ -0,0 +1,14 @@
+/*
+
+The contracts package provides Go bindings for our contracts.
+
+
+The bindings are generated with `abigen`. `jq` is required to parse out the ABI section
+from the hardhat artifacts.
+
+*/
+
+package contracts
+
+//go:generate sh -c "jq .abi ../../packages/contracts/artifacts/contracts/L1/DepositFeed.sol/DepositFeed.json | abigen --pkg deposit --out deposit/deposit_feed_raw.go --abi -"
```

Comment:
You can pass in the bytecode like this https://github.com/ethereum-optimism/optimism/blob/0cc660d3b2cf78827d008ea01b51f54fb6e5a708/go/gas-oracle/Makefile#L31 to avoid needing to hardcode in the bytecode in the other file. This will make the binary available on a global like so: https://github.com/ethereum-optimism/optimism/blob/0cc660d3b2cf78827d008ea01b51f54fb6e5a708/go/gas-oracle/bindings/gaspriceoracle.go#L34

Reply:
Nice, I copied the makefile here and it's way nicer.